documents/schemas/configs/parsing.json

```diff
@@ -96,6 +96,20 @@
       "default": "console",
       "description": "Error reporter type"
     },
+    "log_level": {
+      "type": "string",
+      "source": "env",
+      "env_var": "LOG_LEVEL",
+      "default": "INFO",
+      "description": "Logging level"
+    },
+    "http_port": {
+      "type": "int",
+      "source": "env",
+      "env_var": "HTTP_PORT",
```

Comment:
Fixed: standardized `http_port.env_var` from `PORT` to `HTTP_PORT` for consistency across services. Parsing service now reads `HTTP_PORT`.